packages/react-server/core/TritonAgent/Request.js

```diff
@@ -101,6 +102,27 @@ Request.prototype.type = function (type) {
 	return this;
 }
 
+Request.prototype.noCacheResponse = function () {
+	this._noCacheResponse = true;
+	return this;
+}
+
+Request.prototype.getCacheEntry = function() {
+	if(this._noCacheResponse) {
+		return new CacheEntry(
+			null,
```

Comment:
Because we don't want it to go into the cache!

This is to support out-of-band requests:  I make a request whose response I'm going to stash away and re-use a bunch of times.

When I make the actual request I'll do:

```javascript
TritonAgent.get(url).noCacheResponse().then(stuffIntoCache);
```

Then, later, when I'm _using_ the response I'll do:

```javascript
TritonAgent.get(url).getCacheEntry().setResponse(getFromCache()).requesters++;
```